### ISSUE TITLE
Bug in learning rate?

### ISSUE THREAD

**Issue by ghost** [fcomitani/simpsom#30]
Should the _update_learning_rate() in network.py be `self.learning_rate = self.start_learning_rate * self.xp.exp(-n_iter / self.epochs)` instead? Thanks!

**Comment by fcomitani** [comment]
Hi @wshinigamic, nice catch. I'll fix it in the next commit.

**Comment by fcomitani** [comment]
Fixed by [63490f6](https://github.com/fcomitani/simpsom/commit/63490f6502f6a114d70b5954407ab9701a1f6abf)
